src/scopes/activityReport/index.test.js

```diff
@@ -625,4 +632,15 @@ describe('filtersToScopes', () => {
         .toEqual(expect.arrayContaining([excludedReport.id, globallyExcluded.id]));
     });
   });
+
+  describe('defaultScope', () => {
+    it('excludes deleted reports', async () => {
+      await ActivityReport.create({ ...reportObject });
+      const beginningAR = await ActivityReport.findAll();
+      expect(beginningAR.length).toEqual(2);
```

Comment:
This feels like a brittle test. Instead of comparing the count here to 2, how about getting the count here, and then making sure it didn't change after creating the deleted report?